cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,35 @@
 /// <reference types='cypress' />
 
-describe('', () => {
+import HomeAndCataloguePageObject from '../support/pages/homeСatalogue.pageObject';
+import ContactFormPageObject from '../support/pages/contactForm.pageObject';
+
+const homeAndCatalogue = new HomeAndCataloguePageObject();
+const contactForm = new ContactFormPageObject();
+
+
+describe('ergeg', () => {
   before(() => {
+    cy.visit('/');
 
   });
 
-  it('', () => {
+  it('should do something< a hipe i do nof forgott ', () => {
+    homeAndCatalogue.clickLaptopBtn();
+    homeAndCatalogue.clickLaptopNameBtn();
+    homeAndCatalogue.clickAddToCartBtn();
+    homeAndCatalogue.click_window_ok_btn();
+    homeAndCatalogue.clickcartBtn();
+    homeAndCatalogue.assertCartPage();
 
+    contactForm.clickPlaceOrderBtn();
+    contactForm.nameField.type(username);
```

Comment:
The variable 'username' is not defined. Please define it or pass a valid string.
You can create testData object in this file with randomly generated data what you will use below